rollup.config.js

```diff
@@ -7,8 +9,11 @@ export default {
   external: Object.keys(pkg.dependencies),
   plugins: [
     babel({
-      runtimeHelpers: true
-    })
+      plugins: ['external-helpers'],
+      externalHelpers: true
+    }),
+    nodeResolve(),
+    commonjs()
```

Comment:
I need it for `import Symbol from 'es6-symbol/polyfill'`. If not, the transformer tricked as external